sql/moz-fx-data-shared-prod/contextual_services_derived/adm_forecasting_v1/query.sql

```diff
@@ -0,0 +1,548 @@
+-- Query for contextual_services_derived.adm_forecasting_v1
+            -- For more information on writing queries see:
+            -- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
+-- DAU SHARE BY COUNTRY
+WITH client_counts AS (
+  SELECT
+    country,
+  -- want qualified desktop clients, any mobile clients
+    (
+      CASE
+      WHEN
+        normalized_app_name = "Firefox Desktop"
+        AND active_hours_sum > 0
+        AND uri_count > 0
+      THEN
+        'desktop'
+      WHEN
+        normalized_app_name != "Firefox Desktop"
+      THEN
+        'mobile'
+      ELSE
+        NULL
+      END
+    ) AS device,
+    submission_date,
+    count(*) AS total_clients,
+    count(
+      CASE
+  -- FIREFOX DESKTOP ELIGIBILITY REQUIREMENTS
+      WHEN
+        normalized_app_name = "Firefox Desktop"
+        AND (
+        -- tiles default on
+          (
+            submission_date >= "2021-09-07"
+            AND browser_version_info.major_version > 92
+            AND country IN UNNEST(
+              ["AU", "BR", "CA", "DE", "ES", "FR", "GB", "IN", "IT", "MX", "US"]
+            )
+          )
+          OR
+        -- Japan experiment now default on
+          (
+            submission_date >= "2022-01-25"
+            AND browser_version_info.major_version > 92
+            AND country = "JP"
+          )
+        )
+      THEN
+        1
+  -- ANDROID ELIGIBLITY REQUIREMENTS
+      WHEN
+        normalized_app_name != "Firefox Desktop"
+        AND normalized_os = "Android"
+        AND browser_version_info.major_version > 100
+      THEN
+        1
+  -- iOS ELIGIBLITY REQUIREMENTS
+      WHEN
+        normalized_app_name != "Firefox Desktop"
+        AND normalized_os = "iOS"
+        AND browser_version_info.major_version > 101
+      THEN
+        1
+      ELSE
+        NULL
+      END
+    ) AS eligible_clients
+  FROM
+    `mozdata.telemetry.unified_metrics`
+  WHERE
+    `mozfun`.bits28.active_in_range(days_seen_bits, 0, 1)
```

Comment:
```suggestion
    mozfun.bits28.active_in_range(days_seen_bits, 0, 1)
```
No need for backticks if project name does not contain dashes.